css/style.css

```diff
@@ -413,44 +403,41 @@ a {
 }
 
 .works-details {
-  max-width: 100%;
   display: grid;
   grid-template-columns: repeat(3, 1fr);
-  gap: 10px;
-  align-items: center;
-  order: 1;
-  margin-top: 30px;
+  grid-template-rows: repeat(2);
+  row-gap: 8px;
   list-style: none;
-  justify-content: space-around;
+  margin: 24px;
+  justify-items: center;
+  align-items: center;
 }
```

Comment:
[ Required ] As suggested by a previous reviewer the `Languages` `Frameworks` and `Skills` sections of the design should be centered as illustrated in the Figma template. You can achieve this by applying the `CSS` below to the `.work-details` class. Please see the screenshot below for more details on this.
```CSS
.works-details {
    display: flex;
    flex-wrap: wrap;
    row-gap: 8px;
    list-style: none;
    margin: 24px;
    justify-content: center;
    align-items: center;
}
```

## Figma template
![Skillscreenshot from 2022-11-03 22-48-29](https://user-images.githubusercontent.com/16392046/199820254-b4699541-bbda-4c5e-b177-0a06f5ca56f8.png)

## Your current implementation
![Screenshot from 2022-11-03 22-46-52](https://user-images.githubusercontent.com/16392046/199819792-c45ca127-240d-4563-afd2-9ab71d6fc158.png)